.github/workflows/auto-merge-successful-prs.yaml

```diff
@@ -0,0 +1,29 @@
+name: Auto-merge Successful PRs
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "*/15 * * * *"  # Every 15 minutes
```

Comment:
not sure if we really need to check it that often (IMHO every day would totally be enough), but I'm also fine with this.